### PR TITLE
Feat/ feature editing

### PR DESCRIPTION
**Pull request by Ramotka**

### CODE REVIEW

File: projects/feature/src/lib/adapters/primary/components/edit-feature.component.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            feature.description,
+            Validators.required
+          ),
+          id: new FormControl(feature.id),
+        });
+      })
+    );
+
+  constructor(
+    @Inject(SELECTS_FEATURE_ID_CONTEXT)
```
Comment (gradzio) [inline projects/feature/src/lib/adapters/primary/components/edit-feature.component.ts:51]: you should not have access to neither DTO nor Context Ports in component. If you use state, you should be consistent and use state all the way. So please Move these dependencies into state and use Commands and Querries

File: projects/feature/src/lib/adapters/primary/components/feature-list.component.html [hunk 2]
```diff
@@ -1,8 +1,69 @@
 <ul *ngIf="featureListQuery$ | async as list" class="my-10">
   <li
     *ngFor="let feature of list.items"
-    class="flex flex-col items-center bg-indigo-50 w-96 p-5 rounded-md my-3 font-medium"
+    class="flex flex-col bg-indigo-50 w-[50rem] p-5 rounded-md my-3"
   >
-    {{ feature }}
+    <!-- <div class="flex justify-between"> -->
```
Comment (gradzio) [inline projects/feature/src/lib/adapters/primary/components/feature-list.component.html:6]: Please no comments in final PR :)

File: projects/feature/src/lib/adapters/primary/resolvers/feature-id.resolver.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import { Observable } from 'rxjs';
+import {
+  SETS_STATE_FEATURE_ID_CONTEXT,
+  SetsStateFeatureIdContextPort,
+} from '../../../application/ports/secondary/context/sets-state-feature-id.context-port';
+
+@Injectable()
+export class FeatureIdResolver implements Resolve<void> {
+  constructor(
+    @Inject(SETS_STATE_FEATURE_ID_CONTEXT)
```
Comment (gradzio) [inline projects/feature/src/lib/adapters/primary/resolvers/feature-id.resolver.ts:16]: You should use Command Port here

**Review by gradzio — CHANGES_REQUESTED** [review]

File: projects/feature/src/lib/adapters/primary/components/edit-feature.component.ts (hunk 1 above)
Comment (Ramotka) [reply projects/feature/src/lib/adapters/primary/components/edit-feature.component.ts:51]: fixed in the following commits

File: projects/feature/src/lib/adapters/primary/components/feature-list.component.html (hunk 2 above)
Comment (Ramotka) [reply projects/feature/src/lib/adapters/primary/components/feature-list.component.html:6]: ok :)

File: projects/feature/src/lib/adapters/primary/resolvers/feature-id.resolver.ts (hunk 3 above)
Comment (Ramotka) [reply projects/feature/src/lib/adapters/primary/resolvers/feature-id.resolver.ts:16]: fixed in the following commits

**Pull request merged**